e2e_playwright/st_write_markdown_test.py

```diff
@@ -42,6 +42,7 @@ def test_displays_markdown(app: Page, assert_snapshot: ImageCompareFunction):
     assert_snapshot(markdown_elements.nth(7), name="write_np_float")
     assert_snapshot(markdown_elements.nth(8), name="write_single_line_monospace_block")
     assert_snapshot(markdown_elements.nth(9), name="write_multi_line_monospace_block")
+    assert_snapshot(markdown_elements.nth(13), name="write_python_code")
```

Comment:
nit: i think its better to just call this `write_inline_code` since its not really python code :)